robolectric/src/main/java/org/robolectric/android/internal/AndroidTestEnvironment.java

```diff
@@ -150,18 +150,20 @@ public void setUpApplicationState(
       loggingInitialized = true;
     }
 
-    SecurityMode.Mode securityMode = configuration.get(SecurityMode.Mode.class);
-    if (securityMode == SecurityMode.Mode.BOUNCY_CASTLE || securityMode == null) {
+    ConscryptMode.Mode conscryptMode = configuration.get(ConscryptMode.Mode.class);
+    Security.removeProvider(CONSCRYPT_PROVIDER);
+    if (conscryptMode == ConscryptMode.Mode.ON || conscryptMode == null) {
```

Comment:
nit: If I understand correctly, we will install Conscrypt provider if user doesn't disable it explicitly. If yes, what about `if (conscryptMode != ConscryptMode.Mode.OFF)` to replace current if statement?

Reply:
I will replace it.